prime-router/src/main/kotlin/azure/QueueAccess.kt

```diff
@@ -34,6 +34,7 @@ class QueueAccess {
     }
 
     fun receiveMessage(queueName: String): Event {
+        // messageText is deprecated
```

Comment:
Same here, if deprecated then change to replacement.

Reply:
Agreed.

Reply:
The replacement is getBody() but this changes the return type from String to BinaryData. Event.parseQueueMessage(message) takes a String as an argument, I think we need to further look into this.

Reply:
@jorg3lopez I would look at how to convert the `BinaryData` into the `String` that `messageText` returns and do that.

Reply:
I'll take a look at it.

Reply:
Follow up: made the changes.